json_to_csv.py

```diff
@@ -70,7 +70,7 @@ def reduce_item(key, value):
         fp = open(json_file_path, 'r')
         json_value = fp.read()
         raw_data = json.loads(json_value)
-
+        fp.close()
```

Comment:
Thanks for catching this. Before I merge it, can you please implement the following suggestion (Add a new line)...

```suggestion
        fp.close()
        
```